scripts/publish_qa.py

```diff
@@ -0,0 +1,39 @@
+import json
+import asyncio
+
+from pyensign.events import Event
+from pyensign.ensign import Ensign
+
+def load_event_fixtures(path):
+    events = []
+    print()
+    with open(path,'r') as f:
+        data = json.load(f)
+        for fixture in data:
+            events.append(Event(json.dumps(fixture["data"]).encode("utf-8"), mimetype=fixture["mimetype"]))
```

Comment:
We will also want to include the `type` (name and version), however it's not possible to do so with the current version of PyEnsign.